package.json

```diff
@@ -18,109 +17,84 @@
   "main": "index.js",
   "license": "Apache-2.0",
   "engines": {
-    "node": ">=v12.22.6"
+    "node": ">=18.15.0"
   },
   "dependencies": {
-    "@alaskaairux/icons": "^4.27.0",
-    "chalk": "^4.1.2",
-    "lit-element": "^2.5.1"
+    "@alaskaairux/icons": "^4.28.0",
+    "chalk": "^5.2.0",
+    "lit": "^2.7.5"
   },
   "peerDependencies": {
-    "@alaskaairux/design-tokens": "^3.3.0",
-    "@alaskaairux/webcorestylesheets": "^3.6.3",
-    "@webcomponents/webcomponentsjs": "^2.4.3",
-    "focus-visible": "^5.1.0"
+    "@alaskaairux/design-tokens": "^3.14.0",
+    "@aurodesignsystem/webcorestylesheets": "^4.6.0"
   },
   "devDependencies": {
-    "@alaskaairux/design-tokens": "^3.11.0",
-    "@alaskaairux/webcorestylesheets": "^3.7.3",
-    "@babel/core": "^7.16.0",
-    "@babel/plugin-syntax-dynamic-import": "^7.8.3",
-    "@babel/plugin-transform-runtime": "^7.16.0",
-    "@babel/preset-env": "^7.16.0",
-    "@babel/runtime": "^7.16.0",
-    "@commitlint/cli": "^13.1.0",
-    "@commitlint/config-conventional": "^13.1.0",
-    "@open-wc/testing": "^2.5.33",
-    "@open-wc/testing-karma": "^4.0.9",
-    "@polymer/iron-demo-helpers": "^3.1.0",
-    "@polymer/iron-icons": "^3.0.1",
-    "@polymer/polymer": "^3.3.1",
-    "@semantic-release/changelog": "^5.0.1",
-    "@semantic-release/git": "^9.0.0",
-    "@semantic-release/npm": "^7.1.3",
-    "@webcomponents/webcomponentsjs": "^2.6.0",
-    "autoprefixer": "^10.4.0",
-    "babel-loader": "^8.2.3",
+    "@alaskaairux/design-tokens": "^3.14.0",
+    "@aurodesignsystem/eslint-config": "^1.3.0",
+    "@aurodesignsystem/webcorestylesheets": "^4.6.0",
+    "@commitlint/cli": "^17.6.5",
+    "@commitlint/config-conventional": "^17.6.5",
+    "@open-wc/testing": "^3.2.0",
+    "@rollup/plugin-node-resolve": "^15.1.0",
+    "@semantic-release/changelog": "^6.0.3",
+    "@semantic-release/git": "^10.0.1",
+    "@semantic-release/npm": "^10.0.4",
+    "@web/dev-server": "^0.2.3",
+    "@web/test-runner": "^0.16.1",
+    "autoprefixer": "^10.4.14",
     "compression": "^1.7.4",
     "concat": "^1.0.3",
-    "copy-webpack-plugin": "^6.4.1",
+    "concurrently": "^8.2.0",
     "copyfiles": "^2.4.1",
-    "core-js": "^3.19.1",
-    "eslint": "^7.32.0",
-    "eslint-plugin-lit": "^1.6.1",
-    "focus-visible": "^5.2.0",
-    "husky": "^4.3.8",
+    "core-js": "^3.31.0",
+    "eslint": "^8.43.0",
+    "eslint-plugin-jsdoc": "^46.2.6",
+    "husky": "^8.0.0",
```

Comment:
v8.0.3 is released. Please ensure that all dependencies are at the lastest version possible.